manifests/bosh-manifest/operations.d/120-cpi.yml

```diff
@@ -11,6 +11,6 @@
   type: replace
   value:
     name: "bosh-aws-cpi"
-    version: "99"
-    url: "https://bosh.io/d/github.com/cloudfoundry/bosh-aws-cpi-release?v=99"
-    sha1: "ffc4a06d6728d88eb108418f886f46428c2a1bf2"
+    version: "101"
```

Comment:
You missed the comment above this section where it tells you to update the https://github.com/alphagov/paas-docker-cloudfoundry-tools/blob/main/bosh-cli-v2/Dockerfile file first.

Should we put in an # IMPORTANT!! there to make it more obvious?